packages/cli/src/commands/remove/command.ts

```diff
@@ -0,0 +1,45 @@
+import { Command } from '../help';
+import { getPkgName } from '../../util/pkg-name';
+
+export const removeCommand: Command = {
+  name: 'remove',
+  description: 'Remove a deployment by name or id',
```

Comment:
This should probably end with a period.

```suggestion
  description: 'Remove a deployment by name or id.',
```

Reply:
@cb1kenobi I don't think this was a blocking comment

Reply:
Correct, however consistency is king.